### PR TITLE
Add scrapers to pull Budget data from OMB

### LINKED ISSUE #15: Add scrapers to pull Budget data from OMB
Follow on issue created to finish the work raised in #14.

The budget data pipeline is currently completely manual. Automating it would make this repo easier to distribute to users, allowing them to pull their own data, rather than maintainers pushing excel files to the repo.

### PR DESCRIPTION
**Pull request by connorjoleary**
closes #15

### CODE REVIEW

**Comment by acxz** [comment]
whoa this is amazing, thx for the contribution @connorjoleary . I'm gonna take a look at this later today or tomorrow and test it out. We're excited you want to help out on this little project!

btw if you plan on continuing to contribute we can get into a call or something so that we can make a little roadmap for this project

**Comment by connorjoleary** [comment]
Sounds great to me!

**Comment by sseshan7** [comment]
@connorjoleary wow, thank you for contributing!

A clean solution to this problem. I'll test this in my server environment this weekend and give some feedback. Thanks again.

**Comment by connorjoleary** [comment]
Thank you guys for the kind words. I'd be happy to meet with you and discuss where you see this going and what I can do to help. My email is connor.oleary24@gmail.com if you want to reach out and set up a time to chat. I'm also working on a project analysing what members of Congress say during congressional hearing and I'd be happy to somehow incorporate that into this project.

**Review by acxz — APPROVED** [review]
Just tested the script, it works wonderfully.

We will need to have some follow up commits to
- use the newly named directory
- remove the existing data and the git-lfs dep

But on its own this is a nice atomic commit.

@stevesdawg LGTM

File: setup.py [hunk 1]
```diff
@@ -22,6 +22,8 @@
         "requests",
         "united_states_congress",
         "xlrd",
+        "requests",
```
Comment (sseshan7) [inline setup.py:25]: A nitpick, remove this line as we already list `requests` as a dep.

**Review by sseshan7 — CHANGES_REQUESTED** [review]
Tiny nitpick change requested, but otherwise, this looks good to me! Thanks for the great feature. I will rebase and merge once you follow up.

**Comment by acxz** [comment]
> My email is [connor.oleary24@gmail.com](mailto:connor.oleary24@gmail.com) if you want to reach out and set up a time to chat.

Do you have a [matrix](https://matrix.org/) account? We can much easily communicate on there instead of an email back and forth. My UN is https://matrix.to/#/@1awtumleef:matrix.org feel free to contact me.

**Review by sseshan7 — APPROVED** [review]

**Comment by sseshan7** [comment]
Added the fix myself, and merged your code in @connorjoleary 🚀

**Pull request merged**